src/payment/payment.ts

```diff
@@ -111,6 +111,10 @@ interface AdyenV2Card {
     bigpay_token?: void;
 }
 
+export interface FormattedBrowserInfo {
+    browser_info: BrowserInfo | null;
```

Comment:
`browser_info` should not be `null`. If we have a type specifically to hold `browser_info` it should always contain it.